random_map:
  map:
    obs_num: [0, 50, 55] # num of Triangle, Quadrangle, Pentagon, ... 20~30
    resolution: 0.01
    size_x: 20.0
    size_y: 20.0
    min_width: 0.4
    max_width: 0.5
    min_dis: 1.5
    sensor_rate: 10.0
    sensor_range: 5.0

egeneration_node:
  map:
    obs_num: [0, 25, 25] # num of Triangle, Quadrangle, Pentagon, ... 20~30
    resolution: 0.01
    size_x: 20.0
    size_y: 20.0
    min_width: 0.4
    max_width: 0.5
    min_dis: 1.8
    sensor_rate: 10.0
    sensor_range: 5.0

    
# image_node:
#   map:
#     obs_num: [23, 23, 23] # num of Triangle, Quadrangle, Pentagon, ... 20~30
#     resolution: 0.01
#     size_x: 20.0
#     size_y: 20.0
#     min_width: 0.8
#     max_width: 0.9
#     min_dis: 1.7
#     sensor_rate: 10.0
#     sensor_range: 5.0
# corridor_node:
#   map:
#     obs_num: [23, 23, 23] # num of Triangle, Quadrangle, Pentagon, ... 20~30
#     resolution: 0.01
#     size_x: 20.0
#     size_y: 20.0
#     min_width: 0.8
#     max_width: 0.9
#     min_dis: 1.7
#     sensor_rate: 10.0
#     sensor_range: 5.0